### PR TITLE
Added Card Managment Screen

### PR DESCRIPTION
**Pull request by vijaytembugade**
Added Card Management page For ecommerce.

### CODE REVIEW

**Review by sakshamAK — APPROVED** [review]
great work!!

**Review by horsemaker — APPROVED** [review]
LGTM! 👍

File: Screens/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  flex-wrap: wrap;
+}
+
+.cart-container > .cart-list {
+  flex: 0.6;
+  padding: 3rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: flex-end;
```
Comment (vaibhav18matere) [inline Screens/Cart/cart.css:14]: can make separate class for this in main css file

**Review by vaibhav18matere — APPROVED** [review]

File: Screens/Cart/cart.css (hunk 1 above)
Comment (vijaytembugade) [reply Screens/Cart/cart.css:14]: Thank for pointing it out. On it

**Review by MarkVed17 — APPROVED** [review]
Awesome work, Vijay!
Responsiveness right from the beginning 🔥

**Pull request merged**